src/late_multicellular_stage/MulticellularConvolutionDispayer.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections.Generic;
+using Godot;
+
+/// <summary>
+///   Creature component to display it using convolution surfaces
+/// </summary>
+public partial class MulticellularConvolutionDispayer : MeshInstance3D, IMetaballDisplayer<MulticellularMetaball>
+{
+    private const float AABBMargin = 0.1f;
+
+    private StandardMaterial3D? material;
+
+    private float? overrideColourAlpha;
+
+    public float? OverrideColourAlpha
+    {
+        get => overrideColourAlpha;
+        set
+        {
+            // Due to both being nullable, this would be a bit complicated to compare with an epsilon value
+            // ReSharper disable once CompareOfFloatsByEqualityOperator
+            if (overrideColourAlpha == value)
+                return;
+
+            overrideColourAlpha = value;
+            ApplyAlpha();
+        }
+    }
+
+    public override void _Ready()
+    {
+        base._Ready();
+
+        // This is here in case we need custom shader effects at some point
+        // Material = new ShaderMaterial()
```

Comment:
On reading the full code I now see that this probably was a case where originally I used the current node's material here, then left the code commented out and finally moved to a different approach. This is probably the reason why Material is capitalized here, but it shouldn't really matter so this is just fine to leave like it is currently.